File: test-sbt-project/src/main/scala/MainActivity.scala
package net.kriomant.buketan_sbt_test

import _root_.android.app.Activity
import _root_.android.os.Bundle

class MainActivity extends Activity with TypedActivity {
  override def onCreate(bundle: Bundle) {
    super.onCreate(bundle)
    setContentView(R.layout.main)

    findView(TR.textview).setText("hello, world!")

	  val chatRes = R.drawable.chat
	  val testRes = R.drawable.test
  }
}

File: test-ant-project/src/net/kriomant/buketan/test_ant_project/MainActivity.java
package net.kriomant.buketan.test_ant_project;

import android.app.Activity;
import android.os.Bundle;

public class MainActivity extends Activity
{
    /** Called when the activity is first created. */
    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);

		int chatResource = R.drawable.chat;
		int testResource = R.drawable.test;
    }
}
